Sources/Keyboard/Keyboard.swift

```diff
@@ -132,12 +157,14 @@ extension Keyboard where Content == KeyboardKey {
                 latching: Bool = false,
                 layout: KeyboardLayout = .piano,
                 noteOn: @escaping (Pitch) -> Void = { _ in },
-                noteOff: @escaping (Pitch) -> Void = { _ in }) {
+                noteOff: @escaping (Pitch) -> Void = { _ in },
+                rows: Int = 6){
```

Comment:
I'm not a fan of this being after noteOn/noteOff when it is closely tied to layout.